data/countries/IL.yaml

```diff
@@ -0,0 +1,210 @@
+holidays:
+  # @attrib https://en.wikipedia.org/wiki/Public_holidays_in_Israel
+  IL:
+    names:
+      he: מְדִינַת יִשְׂרָאֵל
+      en: Israel
+    langs:
+      - he
+    zones:
+      - Asia/Jerusalem
+    dayoff: Saturday
+    days:
+      1 Tishrei:
+        name:
+          he: ראש השנה
+          en: Rosh Hashanah (Yom Teruah)
+      2 Tishrei:
+        name:
+          he: ראש השנה יום 2
+          en: Rosh Hashanah (Day 2)
+      3 Tishrei if Saturday then next Sunday:
+        name:
+          he: צום גדליה
+          en: Fast of Gedaliah (Tzom Gedalya)
+        type: observance
+      10 Tishrei:
+        name:
+          he: יום כיפור
+          en: Day of Atonement (Yom Kippur)
+      15 Tishrei:
+        name:
+          he: סוכות
+          en: Feast of Tabernacles (Sukkot)
+      16 Tishrei P6D:
+        name:
+          he: חול המועד סוכות
+          en: The week of Sukkot
+        type: school
+        note: Collective paid leave in many businesses and government offices
+      22 Tishrei:
+        name:
+          he: שמחת תורה/שמיני עצרת
+          en: Simchat Torah/ Shmini Atzeret
+      12 Cheshvan if Friday,Saturday then previous Thursday since 1997:
+        name:
+          he: יום רבין
+          en: Rabin Day
+        type: observance
+        note: National remembrance day, business as usual
+      29 Cheshvan:
+        name:
+          he: סיגד
+          en: Sigd
+        type: observance
+        note: Festival of the Ethiopian Jews, optional paid leave, business as usual
+      6 Kislev if Saturday then next Sunday:
+        name:
+          en: Ben-Gurion Day
+          he: יום בן-גוריון
+        type: observance
+        note: Day marked by the Knesset
+      25 Kislev P8D:
+        name:
+          en: Hanukkah
+          he: חנוכה
+        type: school
+      10 Tevet:
+        name:
+          en: Tenth of Tevet
+          he: צום עשרה בטבת
+        type: observance
+      15 Shvat:
+        name:
+          en: Tu Bishvat (Arbor Day)
+          he: ט"ו בשבט
+        type: observance
+      13 AdarII if Saturday then previous Thursday:
+        name:
+          en: Fast of Esther
+          he: תענית אסתר
+        type: school
+      14 AdarII P2D:
+        name:
+          en: Purim
+          he: פורים
+        type: school
+        note: optional paid leave
+      10 Nisan:
+        name:
+          en: Aliyah Day
+          he: יום העלייה
+        type: observance
+        note: Official holiday, business as usual
+      15 Nisan:
+        name:
+          en: Passover
+          he: פסח
+      16 Nisan P5D:
+        name:
+          en: Passover (intermediate days)
+          he: חול המועד פסח
+        type: school
+        note: School holiday, collective paid leave in many businesses and government offices
+      21 Nisan:
+        name:
+          en: Seventh day of Passover
+          he: שביעי של פסח
+      22 Nisan:
```

Comment:
it should be 21 Nisan

Reply:
I found:
- Simchat Torah: 22nd (outside of Israel 23rd) day of Tishrei
- Mimouna: begins 21st day of Nisan in Israel; 22nd day of Nisan outside of Israel

so logically either both should be according to the date in Israel or both according to the date outside of Israel.

Reply:
However, the remaining holidays in this file look good to me.

Reply:
Beside Wikipedia, I found:
- https://www.theshulofnewyork.org/event/passover-mimouna-2019.html
- https://www.jesusboat.com/all-about-mimouna-post-passover/
- https://www.sukkatshalom.org/event/end-of-passover-moroccan-mimouna.html